app/Handlers/Events/SendFirstVoteMessage.php

```diff
@@ -39,21 +35,38 @@ public function handle(UserCastFirstVote $event)
             'candidate_name' => $event->candidate->name,
 
```

Comment:
`candidate_id => $event->candidate->id` is on the line above. Is there somewhere else I should be adding it in the payload?